Assets/Scripts/Bullets/BulletView.cs

```diff
@@ -22,7 +22,7 @@ private void OnCollisionEnter(Collision collision)
         {
             if (collision.gameObject.tag == "Enemy")
```

Comment:
why don't you directly check if EnemyView is present on this GO to check if its enemy (instead of checking tag) ?

Reply:
Done